components/sbm-recipes-mule-to-boot/src/main/java/org/springframework/sbm/mule/actions/JavaDSLAction2.java

```diff
@@ -54,6 +56,8 @@ public class JavaDSLAction2 extends AbstractAction {
     private final MuleMigrationContextFactory muleMigrationContextFactory;
     private final Map<Class<?>, TopLevelElementFactory> topLevelTypeMap;
 
+    private boolean muleTriggerMeshTransformEnabled;
```

Comment:
While reading myself into the PR again I realize that by using this property the application property could be removed. This would be good as it's not really and application aspect.
We offer two recipes for mule migration instead of requiring a property.